<launch>

   <node pkg="autonomous_landing" name="dense" type="dense_verifier" output="screen" clear_params="true" > <!--launch-prefix="gdb -ex run -deletethis-args" -->
        <remap from="dense/cam" to="/cam_down" />
        <param name="use_tf_transforms" value="true" />
        <param name="poses_supplied" value="true" />
        <param name="cam_frame_topic" value="dense_cam" />
        <param name="world_frame_topic" value="world" />
        <param name="image_scale" value="0.25" />
        <param name="safety_zone_size" value="5.0" />
        <param name="sweep_n" value="6" />
        <param name="window_size" value="7" />
        <param name="occlusion_mode" value="none" />
        <param name="occlusion_best_k" value="5" />
        <param name="matching_threshold" value="0.3" />
        <param name="distance_threshold" value="0.1" />
        <param name="uniqueness_threshold" value="0.95" />
        <param name="normal_smoothing_size" value="3" />
        <param name="max_depth_change_factor" value="200.0" />
        <param name="write_to_file_debug" value="false" />
        <param name="median_filter_kernel_size" value="5" />
        <param name="voxel_size" value="0.3" />
            
        <rosparam file="$(find autonomous_landing)param/midair_down.yaml" command="load" />
            
    </node>

    <node pkg="autonomous_landing" name="detector" type="detector" output="screen" > <!--launch-prefix="gdb -ex run -deletethis-args" -->
        <remap from="detector/input_pointcloud" to="dense/dense_pointcloud" />
        <remap from="detector/input_mesh" to="/voxblox_midair/voxblox_node/mesh" />
        <param name="boxSize" value="2.0" />
        <param name="minPoints" value="16" />
        <param name="voxel_size" value="0.3" />
            
    </node>
       
    
</launch>
